### ISSUE TITLE
unable to POST a new (unqualified) device

### ISSUE THREAD

**Issue by iamdey** [CaliOpen/Caliopen#811]
this is the use case:

1. I've signup with device "default"
2. I signin with a new device "undefined"
3. I save an unqualified device with this payload

```
POST /api/v2/devices
{
	"device_id": "062235ec-19f4-4864-81b8-370ec46b573b",
	"ecdsa_key": {
		"curve": "P-256",
		"hash": "SHA256",
		"x": "1c2f31993dc1efa2e5e083dc66b6f1fb4bbb9437a9bab9b7953a8d903ca13ea0",
		"y": "e9767ef6fbd9c505bb7a9db9d7c415ab2ebd35076df7b81eabf3650c48d7765f"
	}
}
```

For now I get this error:

```
{
	"errors": [{
		"code": 602,
		"message": "device.name in body is required",
		"name": ""
	}, {
		"code": 602,
		"message": "device.type in body is required",
		"name": ""
	}, {
		"code": 616,
		"message": "device.ecdsa_key in body is a forbidden property",
		"name": ""
	}]
}
```

I should be able to pass the almost same payload as in signup /cc @gdchamal 
(ecdsa_key included)

**Comment by sapiens-sapide** [comment]
Need to discuss the difference between the `default device` and a new `device`.

**Comment by sapiens-sapide** [comment]
route no longer available after PR #816
